Core/clim-core/presentation-translators.lisp

```diff
@@ -105,7 +105,7 @@ and used to ensure that presentation-translators-caches are up to date.")
                         old))
   (invalidate-translator-caches)
   (setf (gethash (name translator) (translators table)) translator)
-  (push translator (gethash (from-type translator)
+  (push translator (gethash (presentation-type-name (from-type translator))
```

Comment:
Can the recently added tests for adding and removing presentation translators be extended to verify this fix?

Reply:
this hash table is removed in a later commit, so this is added only for the change atomicity, I don't think there is a need for testing a removed code

Reply:
I see.